### PR TITLE
[plugin.video.ipcams] v0.0.3

### PR DESCRIPTION
**Pull request by b-jesch**
### Description
This addon shows the streams of maximum three surveillance cameras of your own. For this, the URLs of the cams must be entered in the setup. If no stream URLs are entered, a corresponding notification message is issued.
### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by b-jesch** [comment]
Continued from here: https://github.com/xbmc/repo-plugins/pull/1141

File: plugin.video.ipcams/addon.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+<addon id="plugin.video.ipcams" name="Surveillance Cameras" version="0.0.3" provider-name="Birger Jesch">
+  <requires>
+    <import addon="xbmc.python" version="2.25.0"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="default.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <platform>all</platform>
+    <summary lang="en">Shows streams of surveillance cameras</summary>
```
Comment (Tadly) [inline plugin.video.ipcams/addon.xml]: Please change all `en` and `de` etc. to
`en_GB`, `de_DE`
as it says [here](http://kodi.wiki/view/addon.xml#Required_elements)

**Comment by Tadly** [comment]
I'm kinda late to the party and missed the previous PR (would have mentioned it than)

There's a new [Layout of the language folders](http://kodi.wiki/view/Language_support#Layout_of_the_language_folders) which everyone should start using.

File: plugin.video.ipcams/default.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    cam  = [__addon__.getSetting('cam1'), __addon__.getSetting('cam2'), __addon__.getSetting('cam3')]
+    loc  = [__addon__.getSetting('loc1'), __addon__.getSetting('loc2'), __addon__.getSetting('loc3')]
+
+
+if mode is '':
+    _atleast = False
+    for i in range(int(__addon__.getSetting('numcams'))):
+        icon = xbmc.translatePath(os.path.join( __path__,'resources', 'lib', 'media', 'ipcam_%s.png' % (i + 1)))
+        _listitem = '%s - %s' %(item[i], loc[i]) if loc[i] != '' else item[i]
+        li = xbmcgui.ListItem(_listitem, iconImage =icon)
```
Comment (enen92) [inline plugin.video.ipcams/default.py]: this one is deprecated please use setArt

**Comment by enen92** [comment]
Looks okay to me apart from the comment above and @Tadly comments

**Comment by b-jesch** [comment]
Will make changes next week - no access to dev equipment/environment. So please stay...

**Pull request closed without merging**